test/functional/index.test.js

```diff
@@ -1264,7 +1264,7 @@ describe('Indexes', function() {
     metadata: {
       requires: {
         topology: ['single', 'replicaset', 'sharded', 'ssl', 'heap', 'wiredtiger'],
-        mongodb: '>=3.0.0'
+        mongodb: '>=3.0.0 <=4.8.0'
```

Comment:
Looks like version `4.9.0-alpha5-148-g3a70e27` now returns code 86 instead of 85 for this use case, figured we might as well keep the test running on versions prior to that. I went with capping it at the minor version just to be sure we don't run into this on newer versions